extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.example.1.json

```diff
@@ -5,18 +5,14 @@
         {
           "xdm:id": "experience1"
         }
-      ],
-      "xdm:decisionEventID": "TNT:activity1:experience1:1597208433",
-      "xdm:decisionScope": "__view__"
```

Comment:
This example demonstrates the simplest use case: only the item identifiers are collected within that mixin